### ISSUE TITLE
Question about MH rejection rule

### ISSUE THREAD

**Issue by violet-sto** [zdhNarsil/EB_GFN#3]
Hi,

The code for compute MH is: update_dist = torch.distributions.Bernoulli(logits=lp_update + delta_logp_traj) 

Is this inconsistent with the formula p=min(1, exp(lp_update+delta_logp_traj))
